scripts/native.proj

```diff
@@ -71,7 +71,7 @@
     </Exec>
     <!-- Build GitExtensionsShellEx project, x64 -->
     <Exec
-        Condition="'$(Arm64Build)' != 'true'"
+        Condition="'$(TargetPlatform)' != 'arm64'"
```

Comment:
Should this be constrained to x64 only? That is, this can't be used on x86 platforms.

Reply:
The default `x86` package has always included this 64-bit component during build, but it is restricted (i.e. ignored) at installation time, by this condition in `Product.wxs`:

https://github.com/gitextensions/gitextensions/blob/b4100e3222b92bc89d31d9e05084c5584d729cb7/Setup/Product.wxs#L246

So it's safe to keep this thing as-is.

Reply:
I don't think anyone does these days :)